unit_test/sparse/Test_Sparse_spmv.hpp

```diff
@@ -22,6 +22,32 @@ typedef Kokkos::Experimental::half_t kokkos_half;
 
 namespace Test {
 
+// Functor checking that the results of SPMV
+// are consistent with a reference sequential
+// implementation of the same operation.
+//
+// Inputs:
+// - _ex_y      the expected result calculated
```

Comment:
@lucbv this is ok for tests but please follow https://github.com/kokkos/kokkos-kernels/blob/develop/docs/developer/contrib.rst. Following the comment style consistently will help all contributors.